2019/December/README.md

```diff
@@ -1 +1,33 @@
-# talk links and description
+# December Python Pune meetup 2019
+  * [Event page on meetup.com](https://www.meetup.com/PythonPune/events/267036137/)
+  * [Community Newsletter](./community_news.md)
+
+## PythonPune's 5 years journey
+  * **Speaker**: Chandan Kumar
+  * **Description**: Chandan and PythonPune members share their
+    experiences of organizing the meetups consistently over last 5
+    years.
```

Comment:
Here is the link https://gist.github.com/raukadah/573177ca5470caea5a327f5f1a51707a Can we include that? Rest looks good.

Reply:
Oops, sorry I totally forgot about that.

Reply:
Added the link, can you please check it once.